watsonsdk/stt/STTConfiguration.m

```diff
@@ -102,6 +102,7 @@ - (NSString *)getStartMessage{
     [inputParameters setValue:self.interimResults forKey:@"interim_results"];
     [inputParameters setValue:self.continuous forKey:@"continuous"];
     [inputParameters setValue:self.inactivityTimeout forKey:@"inactivity_timeout"];
+    [inputParameters setValue:self.learningOptOut forKey:@"x-watson-learning-opt-out"];
```

Comment:
https://github.com/watson-developer-cloud/speech-ios-sdk/pull/33/commits/4b20393a83d541c6186db1db11fb7a47038b0e52, https://github.com/watson-developer-cloud/speech-ios-sdk/pull/33/commits/079e6c0785cb34c62f4d0ed13ac7257020373dc3, and https://github.com/watson-developer-cloud/speech-ios-sdk/pull/33/commits/c8e79bb7cdfb02a752e6c2d8c929cc9e326968ef are for TTS and STT, and there are more changes in pull request https://github.com/watson-developer-cloud/speech-ios-sdk/pull/33